source/franka_lightweight_interface/examples/joint_velocity_controller.cpp

```diff
@@ -0,0 +1,47 @@
+#include "network_interfaces/control_type.h"
```

Comment:
This file is typically what we would put in another docker to control the franka-lwi by sending it velocity commands ?

Reply:
yes correct